wallet/lib/pages/transaction_failure_manager/local_transaction_list_screen.dart

```diff
@@ -0,0 +1,169 @@
+              child: SvgPicture.asset(SVGUtil.TRANSACTION_RETRY, height: 12.h),
+            ),
+            SvgPicture.asset(SVGUtil.TRANSACTION_FAILED, height: 15.h),
+          ],
+        );
+      case TransactionStatus.Undefined:
+        return const Icon(Icons.error_outline, color: Colors.redAccent);
+    }
+  }
+
+  Widget getMyDateStamp({required int txTime}) {
+    final DateTime date = DateTime.fromMillisecondsSinceEpoch(txTime);
+    return Padding(
+      padding: EdgeInsets.all(5.h),
+      child: Text("${monthStrMap[date.month]!} ${date.day.toString()}",
+          style: TextStyle(
+            color: kDarkGrey,
+            fontSize: 12.sp,
+          )),
+    );
+  }
+
+  Widget getTransactionTitle({required LocalTransactionModel txModel}) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: 6.w),
+          child: Text(
+            txModel.transactionDescription.trimStringMedium(stringTrimConstantMid),
+            style: TextStyle(
+              color: kBlack,
+              fontFamily: kUniversalFontFamily,
+              fontWeight: FontWeight.w800,
+              fontSize: 12.sp,
+            ),
+          ),
+        ),
+        const Spacer(),
+        SizedBox(width: 40.w, child: getTransactionStatusButton(txManager: txModel)),
+      ],
+    );
+  }
+
+  Widget getTransactionTrailings({required LocalTransactionModel txModel}) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        Text("${txModel.transactionPrice}  ${txModel.transactionCurrency}",
+            style: TextStyle(
+              color: kBlack,
+              fontFamily: kUniversalFontFamily,
+              fontWeight: FontWeight.w800,
+              fontSize: 11.sp,
+            )),
+        Icon(Icons.keyboard_arrow_right_rounded, size: 20.r)
+      ],
+    );
+  }
+
+  Widget getEmptyListWidget() {
+    return const Center(
+      child: Text(
+        "No Failures!!",
+        style: TextStyle(color: Colors.black),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: SingleChildScrollView(
+      child: Column(
+        children: [
+          SizedBox(height: 60.h),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              IconButton(onPressed: () => Navigator.pop(context), icon: Icon(Icons.arrow_back_ios_sharp, size: 25.r)),
+              Text(
+                'all_transactions'.tr(),
+                style: TextStyle(color: kBlack, fontFamily: kUniversalFontFamily, fontWeight: FontWeight.bold, fontSize: 20.sp),
+              ),
+              SizedBox(width: 40.w),
+            ],
+          ),
+          Padding(
+            padding: EdgeInsets.symmetric(horizontal: 10.w),
+            child: ChangeNotifierProvider<FailureManagerViewModel>.value(
```

Comment:
Create a content widget for this class. and instead of using change notifier here. Change it from the top. See how screens like general, practice test are made.

Reply:
Alright, thanks.

Reply:
Updated. Please have a look.